components/ledger/pkg/storage/ledgerstore/accounts.go

```diff
@@ -21,18 +22,35 @@ const (
 	accountsTableName = "accounts"
 )
 
+// This regexp is used to validate the account name
+// If the account name is not valid, it means that the user putted a regex in
+// the address filter, and we have to change the postgres operator used.
+var accountNameRegex = regexp.MustCompile(`^[a-zA-Z_0-9]+$`)
+
 type Accounts struct {
 	bun.BaseModel `bun:"accounts,alias:accounts"`
 
-	Address  string            `bun:"address,type:varchar,unique,notnull"`
-	Metadata map[string]string `bun:"metadata,type:jsonb,default:'{}'"`
+	Address     string            `bun:"address,type:varchar,unique,notnull"`
+	Metadata    map[string]string `bun:"metadata,type:jsonb,default:'{}'"`
+	AddressJson []string          `bun:"address_json,type:jsonb"`
```

Comment:
should be account_json no?

Reply:
No, it's address_json to be in accordance with the address field !